python/tvm/relay/frontend/onnx.py

```diff
@@ -951,6 +1024,16 @@ def from_onnx(self, graph, opset):
                 else:
                     dtype = d_type
                 self._nodes[i_name] = new_var(i_name, shape=tshape, dtype=dtype)
+        # get list of unsupported ops
+        unsupported_ops = []
+        for node in graph.node:
+            op_name = node.op_type
+            if op_name not in supported_ops:
```

Comment:
checking op_name in identity_list or op_name in convert_map or op_name = 'Constant' should be enough